src/ResourceManager/IotHub/Commands.IotHub.Test/ScenarioTests/IotHubTests.cs

```diff
@@ -39,5 +39,17 @@ public void TestAzureIotHubLifeCycle()
                 TestUtilities.GetCallingClass(),
                 TestUtilities.GetCurrentMethodName());
         }
+
+        [Fact]
+        [Trait(Category.AcceptanceType, Category.CheckIn)]
+        public void TestAzureIotHubCertificateLifeCycle()
+        {
+            IotHubController.NewInstance.RunPsTestWorkflow(
+                () => { return new[] { string.Format("{0} {1} {2} {3} {4}", "Test-AzureRmIotHubCertificateLifecycle", "northeurope", "pstestiothub", "pstestrg", "S1") }; },
```

Comment:
This is similar to the existing test

Reply:
this is similar to existing test pattern

Reply:
Yes, but making them dynamic will be helpful for anyone who wants to rerecord this test in the future.  If a user wants to record the test on a non-AzureCloud account, it will immediately fail.